src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/SideRail.razor

```diff
@@ -0,0 +1,8 @@
+﻿<div class="component-siderail">
+    <ul class="link-container">
+        @foreach (var param in SideRailParameters)
+        {
+            <li><BitLink Class="@param.Class" OnClick="@(() => ScrollToFragment(param))">@param.Title</BitLink></li>
```

Comment:
plz correct the code formatting in this line and put the elements in separate lines